authentication/pipeline/compliance.py

```diff
@@ -63,7 +63,7 @@ def verify_exports_compliance(
             log.exception(
                 "Exception sending email to support regarding export compliance check failure"
             )
-        raise UserExportBlockedException(backend)
+        raise UserExportBlockedException(backend, export_inquiry.reason_code)
```

Comment:
What are the potential values for `reason_code` ?

Reply:
In theory - 200, 700, 701, 702, 703